resources/src/config.php

```diff
@@ -5,7 +5,7 @@ class ProjectConfig
     public $config;
     public function __construct()
     {
-        $creds = json_decode(file_get_contents($_SERVER['DOCUMENT_ROOT'].'/dbcreds.json'), true);
+        $creds = json_decode(file_get_contents($_SERVER["DOCUMENT_ROOT"]."/../resources/dbcreds.json"), true);
```

Comment:
And also that you have changed the dbcreds from root to resources

Reply:
I have not moved `dbcreds`, I have just changed its path